### PR TITLE
Remove Tag from Exposures

### PR DESCRIPTION
**Pull request by emanuel-schmid**
Changes proposed in this PR:
- removes tag attribute from Exposures and subclasses
- introduce new attribute of LitPop: `description`, which is used in `plot_hexbin` and `plot_scatter` to set the title

This PR fixes #

### PR Author Checklist

- [ ] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Descriptive pull request title added
- [x] Source branch up-to-date with target branch
- [x] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#2.--Commenting-&-Documenting) updated
- [x] [Tests][testing] updated
- [x] [Tests][testing] passing
- [x] No new [linter issues][linter]
- [x] [Changelog](https://github.com/CLIMADA-project/climada_python/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/stable/guide/Guide_Continuous_Integration_and_Testing.html#3.C.--Static-Code-Analysis

### CODE REVIEW

File: climada/entity/exposures/base.py [hunk 1]
```diff
@@ -546,17 +543,19 @@ def plot_scatter(self, mask=None, ignore_zero=False, pop_name=True,
...
 
         Returns
         -------
         cartopy.mpl.geoaxes.GeoAxesSubplot
         """
         crs_epsg, _ = u_plot.get_transformation(self.crs)
-        title = "\n".join(self.tag.description)
-        cbar_label = f'Value ({self.value_unit})'
+        title = (self.description.strip() if isinstance(self.description, str) else ""
+                 ) if title is None else title
```
Comment (peanutfun) [inline climada/entity/exposures/base.py:558]: `title = something if title is None else title` seems redundant to me. Also: What other type than `str` would `description` support? Should we maybe cast it to `str` in the constructor to avoid testing its type here?
```suggestion
        if title is None:
            title = self.description.strip()
```

File: climada/entity/exposures/base.py [hunk 2]
```diff
@@ -615,8 +621,8 @@ def plot_hexbin(self, mask=None, ignore_zero=False, pop_name=True,
...
```
Comment (peanutfun) [inline climada/entity/exposures/base.py:625]: Same as above. `title` is already assigned to `title` 😉

File: climada/engine/forecast.py [hunk 3]
```diff
@@ -1079,18 +1079,6 @@ def _plot_warn(
...
-            & isinstance(decision_dict["area_aggregation"], float)
-        ):
-            ValueError(
-                " If decision_level is 'exposure_point',"
-                + "parameters probability_aggregation and "
-                + "area_aggregation of "
-                + "Forecast.plot_warn_map() must both be "
-                + "floats between [0..1]. Which each "
-                + "specify quantiles."
-            )
```
Comment (peanutfun) [inline climada/engine/forecast.py:1093]: How is this related to the Exposures tag?

**Review by peanutfun — CHANGES_REQUESTED** [review]
Two minor issues. The rest looks well! 👌

File: climada/entity/exposures/base.py (hunk 1 above)
Comment (emanuel-schmid) [reply climada/entity/exposures/base.py:558]: `None` is the default. So if you call it with a `title`-argument - fine, up to you, even if it's an empty string.
If you don't bother - also fine, then the title from the description is plotted.

Comment (emanuel-schmid) [reply climada/entity/exposures/base.py:558]: your suggestion doesn't work if the `description` is `None`. You would need another `if` block. From there on it's a question of taste I guess.

Comment (peanutfun) [reply climada/entity/exposures/base.py:558]: Then let's keep it as simple as can be: Users need to take care of formatting their descriptions. So I suggest we just do:
```suggestion
    if title is None:
        title = self.description
```
This should also work when `description` is `None`, `ax.set_title(None)` should have no (negative) effect.

File: climada/engine/forecast.py (hunk 3 above)
Comment (peanutfun) [reply climada/engine/forecast.py:1093]: Forecast will be overhauled anyway. I thinks it's best not to touch that class now. Can you revert the changes, even though the error clearly should be raised? 😅

File: climada/entity/exposures/base.py (hunk 1 above)
Comment (emanuel-schmid) [reply climada/entity/exposures/base.py]: some util.plot function doesn't like `None`. Hence I did `tjitle = self.description or None`.
